### PR TITLE
try to disable questionnaire-button on status COMPLETE

### PR DESCRIPTION
**Pull request by eilivjl**
load questionnairestatus from firebase, if complete, disable navigation to questionnaire

### CODE REVIEW

File: src/pages/home.tsx [hunk 1]
```diff
@@ -48,6 +52,22 @@ const Home = () => {
 
     console.log('hiddenComponent', hiddenUploadComponent);
 
+    useEffect(() => {
```
Comment (prabu-ssb) [inline src/pages/home.tsx:55]: Is this not loaded up at login time in the current impl ? 
Why load it up again in home ?

File: src/pages/home.tsx [hunk 2]
```diff
@@ -74,15 +94,27 @@ const Home = () => {
                         setShowAddExpensesDialog(true);
                     }}
                 />
+                {questionnaireStatus === 'COMPLETE' &&
                 <HomeCTA
                     text={t('questionnaire.title')}
                     styleClass={styles.questionnaireCTA}
+                    onClick={() => {
```
Comment (prabu-ssb) [inline src/pages/home.tsx:101]: I would add a `disabled` or `completed` styleClass and style it differently. I would also replace the arrow with something that shows a completed status. 

it is not good UI to display a clickable entity but have no action associated.

File: src/pages/home.tsx (hunk 1 above)
Comment (eilivjl) [reply src/pages/home.tsx:55]: it is not loaded in in the current impl

Comment (eilivjl) [reply src/pages/home.tsx:55]: but I agree that it should be loaded in at login time

File: src/pages/home.tsx (hunk 2 above)
Comment (eilivjl) [reply src/pages/home.tsx:101]: agreed

**Comment by eilivjl** [comment]
Changes to load questionnaire status on login. Set questionnaire status to update in app when updated in firebase. A few fixes on questions

**Pull request merged**